.github/workflows/ros2-ci.yaml

```
@@ -9,37 +9,44 @@ on:
 jobs:
 
   build-ros2:
-    runs-on: ubuntu-latest
+    runs-on: [ubuntu-latest]
     strategy:
       fail-fast: false
       matrix:
-        ros-version: [foxy, galactic]
+        ros_distribution:
+          - foxy
+          - galactic
+          - humble
+          - rolling
         include:
-          - ros-version: foxy
-            image: ros:foxy
-          - ros-version: galactic
-            image: ros:galactic
+          # Foxy Fitzroy (June 2020 - May 2023)
+          - docker_image: rostooling/setup-ros-docker:ubuntu-focal-ros-foxy-ros-base-latest
+            ros_distribution: foxy
+            ros_version: 2
+          # Galactic Geochelone (May 2021 - November 2022)
+          - docker_image: rostooling/setup-ros-docker:ubuntu-focal-ros-galactic-ros-base-latest
+            ros_distribution: galactic
+            ros_version: 2
+          # Humble Hawksbill (May 2022 - May 2027)
+          - docker_image: rostooling/setup-ros-docker:ubuntu-jammy-ros-humble-ros-base-latest
+            ros_distribution: humble
+            ros_version: 2
+          # Rolling Ridley  (June 2020 - Present)
+          - docker_image: rostooling/setup-ros-docker:ubuntu-jammy-ros-rolling-ros-base-latest
+            ros_distribution: rolling
+            ros_version: 2
```

I guess we can drop the `ros_version` data.

```suggestion
          # Galactic Geochelone (May 2021 - November 2022)
          - docker_image: rostooling/setup-ros-docker:ubuntu-focal-ros-galactic-ros-base-latest
            ros_distribution: galactic
          # Humble Hawksbill (May 2022 - May 2027)
          - docker_image: rostooling/setup-ros-docker:ubuntu-jammy-ros-humble-ros-base-latest
            ros_distribution: humble
          # Rolling Ridley  (June 2020 - Present)
          - docker_image: rostooling/setup-ros-docker:ubuntu-jammy-ros-rolling-ros-base-latest
            ros_distribution: rolling
```